<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Username</class>
 <widget class="QWidget" name="Username">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>791</width>
    <height>455</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QPushButton" name="ButtonCheck">
   <property name="geometry">
    <rect>
     <x>199</x>
     <y>385</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Überprüfe Daten</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Email">
   <property name="geometry">
    <rect>
     <x>101</x>
     <y>187</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lES2">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>320</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lES1">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>250</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labelGreeting">
   <property name="geometry">
    <rect>
     <x>59</x>
     <y>15</y>
     <width>511</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:10pt;&quot;&gt;Du hast deinen Username vergessen? Gar kein Problem!&lt;br/&gt;Gib einfach deine Daten ein und wir zeigen dir deinen Usernamen.&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>224</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sicherheitsfrage 1: Wie lautet dein Geburtsjahr?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>99</x>
     <y>295</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Sicherheitsfrage 2: Wie heißt der Geburtsname deiner Mutter?</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>160</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>E-Mail:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ForgotPassword">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>420</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Passwort vergessen?</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lEUsername">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>380</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
